### DISCUSSION TITLE
收集各学校的适用情况

### DISCUSSION THREAD

**Discussion by Pandaft** [Pandaft/ESurfingPy-CLI#3] [Announcements]
如果你的学校校园网是广东天翼校园网，可以在此留言此项目程序的适用情况。

```
学校名称：_______________
校园网可网页认证登录：是/否
项目程序能否正常使用：是/否
其他备注：_______________
```

以上是模板，其他更多信息认为有必要透露的可以自行增加。

**Comment by Pandaft** [comment]
占楼，不定期更新至仓库文档。

**Reply by wwwlxs** [reply]
学校名称：_______________
校园网可网页认证登录：是/
项目程序能否正常使用：/否
其他备注：_______________mac os 用不了

**Reply by Pandaft** [reply]
> 学校名称：_______________ 校园网可网页认证登录：是/ 项目程序能否正常使用：/否 其他备注：_______________mac os 用不了

进QQ群（文档最下方有群号）联系我（群主）跟进一下看看你的校园网登录过程有什么不同，我看看能否解决问题吧
